sources/models/shared-defaults/aws-host-containers.toml

```diff
@@ -13,3 +13,6 @@ superpowered = false
 [metadata.settings.host-containers.control.source]
 setting-generator = "schnauzer settings.host-containers.control.source"
 template = "{{ ecr-prefix settings.aws.region }}/bottlerocket-control:v0.4.2"
+
+[metadata.settings.host-containers.admin.user-data]
+setting-generator = "shibaken"
```

Comment:
This should probably move up so it comes after `[metadata.settings.host-containers.admin.source]`.